packages/camera/camera_web/example/integration_test/camera_web_test.dart

```diff
@@ -2330,7 +2330,6 @@ void main() {
         when(
           () => cameraService.getMediaStreamForOptions(
             any(),
-            cameraId: cameraId,
```

Comment:
Same as comment above.

Reply:
fixed in [38eaffa](https://github.com/flutter/plugins/pull/6163/commits/38eaffa10e34d442ad98669b697221ea513615a4)